src/engine.rs

```diff
@@ -295,6 +295,13 @@ where
         path: &std::path::Path,
         file_builder: Arc<B>,
     ) -> Result<Vec<(u64, u64)>> {
+        if !path.exists() {
```

Comment:
Why raise error? We create the directory if not exist here:
https://github.com/tikv/raft-engine/blob/8c3e663a4a5af34bd9867489adcb683c6383aa6a/src/file_pipe_log.rs#L442-L445

Reply:
@tabokie 
When we run `clt check --path x` , If the path `x` does not exist and the function will automatically create a directory `x` thats contains raft engine files. I think we should explicitly throw error if the path have not exist in the check logic.

Reply:
Okay, you can do a check in `consistency_check`, but normal `open` behavior should be kept the same.